src/gui/mzroll/tabledockwidget.cpp

```diff
@@ -368,6 +373,25 @@ void TableDockWidget::updateItem(QTreeWidgetItem* item) {
         }
     }
 }
+
+void TableDockWidget::updateCompoundWidget() {
+
+    _mainwindow->ligandWidget->resetColor();
+
+    QTreeWidgetItemIterator itr(treeWidget);
+    while (*itr) {
+        QTreeWidgetItem* item =(*itr);
```

Comment:
You can have this outside, just as stated above
It's your call whether you want to change it now or later